flang/lib/Lower/Bridge.cpp

```diff
@@ -740,7 +750,8 @@ class FirConverter : public Fortran::lower::AbstractConverter {
     llvm::SmallVector<int64_t> indexList;
     llvm::SmallVector<mlir::Block *> blockList;
     int64_t index = 0;
-    for (auto &label : std::get<std::list<Fortran::parser::Label>>(stmt.t)) {
+    for (Fortran::parser::Label label :
+         std::get<std::list<Fortran::parser::Label>>(stmt.t)) {
```

Comment:
@clementval — do you have any advice on this one? The type is clearly indicated in the template. Is the repetition required here?

Reply:
I would think that this one can pass with `auto`. Probably depends on who reviews it.